src/styles/messages.scss

```diff
@@ -878,6 +878,7 @@
 	height: 100%;
 	width: 100%;
 	display: flex;
+	flex-direction: column;
```

Comment:
it has only 1 div as a child, does ir really need flex-direction:column? i think some other css is redundant as well